src/components/TodoError/TodoError.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+import { Errors } from '../../types/Errors';
+
+type Props = {
+  errorType: Errors | null,
+  setErrorType: (error: Errors | null) => void,
+};
+
+export const TodoError: React.FC<Props> = ({
+  errorType,
+  setErrorType,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setErrorType(null);
+    }, 3000);
+  }, [setErrorType]);
```

Comment:
if you want to rerender your component when you get error, it should be errorType in array of dependencies